### PR TITLE
document infer

### LINKED ISSUE #120: Document infer recipe
Lots of unstated or non-obvious requirements for formatting syntax, so this recipe must be well-documented https://github.com/aguirre-lab/ml4c3/issues/1189#issuecomment-765716595.

### PR DESCRIPTION
**Pull request by StevenSong**
closes #120

### CODE REVIEW

File: Recipes.md [hunk 1]
```diff
@@ -135,15 +135,34 @@ The `train` recipe has a large number of input parameters that are explained in
 ### Overview
 Evaluate model performance and save model predictions for inspection.
 ### Input
+1. `--model_file`: Path to a pretrained model.
```
Comment (erikr) [inline Recipes.md:138]:
```suggestion
1. `--model_file`: Path to a trained model `.h5` file.
```

File: Recipes.md [hunk 2]
```diff
@@ -135,15 +135,34 @@ The `train` recipe has a large number of input parameters that are explained in
...
+1. `--remap_layer`: An existing layer name and the new name to give to the layer. Layer names are best found by inspecting architecture diagrams. Input and output layers will have `input` or `output` prefixes and the tensor map interpretation as a suffix e.g. `input_ecg_2500_continuous`. Repeat this argument for each layer to rename.
```
Comment (erikr) [inline Recipes.md:139]:
```suggestion
1. `--remap_layer`: Two-part arg: 1) existing layer name, followed by a space, and then 2) the new name to give to the layer. Layer names are best found by inspecting architecture diagrams. Input and output layers are prefixed by `input_` or `output_`,  and suffixed by tensor map interpretation, e.g. `ecg_2500` becomes `input_ecg_2500_continuous`. Repeat this argument for each layer to rename.
```

**Review by erikr — CHANGES_REQUESTED** [review]
Thank you! Added a few details.

**Review by erikr — APPROVED** [review]
Gracias!

**Comment by StevenSong** [comment]
@epalto ?

**Pull request closed without merging**